tests/foreman/cli/test_activationkey.py

```diff
@@ -1276,6 +1276,55 @@ def test_bugzilla_1111723(self):
 
         self.assertEqual(new_activation_key['name'], name)
 
+    def test_remove_host_by_id(self):
+        """@Test: Test that hosts associated to Activation Keys can be removed
+        using id of that host collection
+
+        @Feature: Activation key - Host
+
+        @Steps:
+
+        1. Create Activation key
+        2. Create hosts collection
```

Comment:
It is a description section, so why not use English language?
Basically, I am referring to UI and common sense